tests/schemas/invalid/test_invalid_hub_and_spoke_and_summary_confirmation_non_existent.json

```diff
@@ -28,6 +28,7 @@
     {
       "id": "employment-section",
       "title": "Employment",
+      "summary": {},
```

Comment:
I don't think we should allow empty summary definitions - at a minimum we should force it to be populated, such as we do with the hub.

```json
{
	"summary": {
		"enabled": true
	}
}
```

would do

Reply:
Completely valid point. I just went with the story description which said they should empty

Reply:
Resolving as this will be followed up in a separate card